src/website/app/demos/layout/components/DemoLayout.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import { clearFix } from 'polished';
+import { createStyledComponent } from '../../../../../styles';
+
+export default createStyledComponent('div', () => ({
+  ...clearFix(),
+
+  '& > *:not(:last-child)': {
+    marginBottom: '1rem'
```

Comment:
it looks like this might need to be taken off more than just the last child in some cases

Reply:
can we remove it from all the children in the last row somehow?

Reply:
Such as...?